Readme.markdown

```diff
@@ -1,195 +1,12 @@
-Google Safe Browsing API
+!! DEPRECATED !! Google Safe Browsing API !! DEPRECATED !!
 ========================
 
-[![Build Status](https://travis-ci.org/rjohnsondev/go-safe-browsing-api.svg)](https://travis-ci.org/rjohnsondev/go-safe-browsing-api)
-[![Coverage Status](https://coveralls.io/repos/rjohnsondev/go-safe-browsing-api/badge.png?branch=master)](https://coveralls.io/r/rjohnsondev/go-safe-browsing-api?branch=master)
+This is an **UNMAINTAINED AND DEPRECATED** fork of
```

Comment:
Fixed in https://github.com/letsencrypt/go-safe-browsing-api/pull/5/commits/47bd6a98ec2f6bf323c1c3d3b9b988c7a20db100